src/modules/verifiable-credentials/types/status-list.types.ts

```diff
@@ -0,0 +1,60 @@
+import { VerifiableCredential } from '@ew-did-registry/credentials-interface';
+
+export interface StatusList2021UnsignedCredential {
+  '@context': Array<string | Record<string, unknown>>;
+  id: string;
+  type: string[];
+  issuer: string;
+  issuanceDate: string;
+  credentialSubject: {
+    id: string;
+    type: string;
+    statusPurpose: string;
+    encodedList: string;
+  };
```

Comment:
Would this make sense?
```suggestion
  credentialSubject: StatusList2021CredentialSubject
```

Reply:
It would, if we move it to `@ew-did-registry/credentials-interface` along with other Status List Credential types

Reply:
Sorry, I'm not really following. Why would moving `StatusList2021UnsignedCredential` to `@ew-did-registry/credentials-interface` impact our ability to make the above change?

Reply:
We improve readability and we can re-use this interface in SSI-Hub for example by storing all related interfaces in one place

Reply:
Ok, no worries, it's a minor point, let's merge as is

Reply:
Okay